tests/ocp/hwol/tests/hwol.go

```
@@ -0,0 +1,570 @@
+			allocatable := node.Object.Status.Allocatable[resName]
+
+			nodePods, err := pod.ListInAllNamespaces(APIClient, metav1.ListOptions{
+				FieldSelector: fmt.Sprintf("spec.nodeName=%s", nodeName),
+			})
+			if err != nil {
+				return false, fmt.Errorf("failed to list pods on node %s: %w", nodeName, err)
+			}
+
+			var requested resource.Quantity
+
+			for _, p := range nodePods {
```

golangci-lint (2.12.2)

[error] variable name 'p' is too short for the scope of its usage

(varnamelen)